torch/storage.py

```diff
@@ -1,11 +1,15 @@
 # mypy: allow-untyped-defs
+
+from __future__ import annotations
```

Comment:
Why do we need to turn on future annotations here? Any particular reason?

Reply:
Sure - quoting works fine. But we could ask the opposite - what's the downside to turning it on?

Reply:
New PEP is considering phasing it out.

Reply:
I assume you're talking about PEP749? From what I can tell it's just phasing out the import statement, not the deferred evaluation aspect that it provides - so sure we'd have to strip the import statement in something like 5 years from now - but the behavior should still be the same.